Palette.js

```diff
@@ -0,0 +1,6 @@
+class Palette {
+  constructor() {
+    this.id = Date.now()
+    this.colors = [new Color(), new Color(), new Color(), new Color(), new Color]
```

Comment:
When a new Palette is created, `this.colors` will now create 5 instances of `colors` in an array.